pbench-dashboard.spec

```diff
@@ -0,0 +1,46 @@
+Name:           {{{ git_name name=pbench-dashboard }}}
+Version:        {{{ git_version lead=3.1 }}}
+Release:        1
+Summary:        The pbench dashboard bits
+
+License:        GPLv3+
+URL:            http://github.com/distributed-systems-analysis/pbench-dashboard
+Source0:        {{{ git_dir_pack }}}
+Buildarch:      noarch
+
+
+BuildRequires: yarn nodejs
+
+%define installdir var/www/html/dashboard
+
+%description
+The pbench "dashboard" UI
+
+%prep
+
+{{{ git_dir_setup_macro }}}
+
+%build
+yarn install
+yarn build
+
+%pre
+getent group pbench > /dev/null || groupadd pbench
+getent passwd pbench > /dev/null || useradd -g pbench -d /home/pbench -c "Pbench user" pbench
+
+%install
+rm -rf %{buildroot}
+mkdir -p %{buildroot}/%{installdir}
```

Comment:
Is it really necessary to do the `rm -rf` here? I worry about doing that with any "unknown" parameter, in general. If you really feel the need to make sure that `%{installdir}` is initially empty, `rm -rf %{buildroot}/%{installdir}` would seem safer since we know that's something we're going to own entirely regardless of the root.

Reply:
My understanding is that the RPM build is (i.e., in the general case) performed in a temporary/restricted environment (e.g., a "chroot"), so the massive `rm` is actually a safe and reasonable thing to do.

Reply:
Your concerns, are valid -- apparently the practice is widespread/standard enough that there is [a warning around using the `--buildroot` switch](http://ftp.rpm.org/max-rpm/ch-rpm-b-command.html#S3-RPM-B-COMMAND-BUILDROOT-WARNING).

(BTW, can you resolve your previous "conversation"?)